SMS2FA.Web/App_Start/IdentityConfig.cs

```diff
@@ -27,37 +30,40 @@ public class SmsService : IIdentityMessageService
 
         public SmsService()
         {
-            _client = new TwilioRestClient(Config.AccountSID, Config.AuthToken);
+            TwilioClient.Init(Config.AccountSid, Config.AuthToken);
         }
 
-        public SmsService(TwilioRestClient client)
+        public SmsService(ITwilioRestClient client)
         {
-            _client = client;
+            TwilioClient.Init(Config.AccountSid, Config.AuthToken);
+            TwilioClient.SetRestClient(client);
         }
 
-        public Task SendAsync(IdentityMessage message)
+        public async Task SendAsync(IdentityMessage message)
```

Comment:
`SMSService` implements `IIdentityMessageService` which is a builtin interface, hence we can't change the return type. It needs to implement it as later the service is used inside a UserManager.
In any case we don't have use for the returned ResourceSet in this code.